ansible/group_vars/all.yml

```
@@ -1,8 +1,11 @@
 ---
 
+arch_info: "{{ 's390x' if 's390x' in test_platform else 'x86'}}"
+
 gcp_ssh_key_file: "{{ lookup('env', 'GCP_SSH_KEY_FILE', default='~/.ssh/google_compute_engine') }}"
+ibmcloud_ssh_key_file: "{{ lookup('env', 'IBMCLOUD_SSH_KEY_FILE', default='~/.ssh/acs-sshkey_rsa.prv') }}"
 
-ansible_ssh_private_key_file: "{{ gcp_ssh_key_file }}"
+ansible_ssh_private_key_file: "{{ ibmcloud_ssh_key_file if arch_info == 's390x' else gcp_ssh_key_file }}"
```

In the interest of making it easier when we want to add ppc64le VMs, maybe we want to check for not x86?
```suggestion
ansible_ssh_private_key_file: "{{ ibmcloud_ssh_key_file if arch_info != 'x86' else gcp_ssh_key_file }}"
```
When the time comes to add arm, this will probably be incredibly messy.